### DISCUSSION TITLE
Windows Binaries

### DISCUSSION THREAD

**Discussion by DJ8014** [bbc/bmx#50] [General]
Hello, I'm sure this is a stupid question, but the main Github page for this repo states "Source and Binary Distributions - Source distributions, including dependencies for the Windows build, **and Windows binaries** are made available in this GitHub repo." I cannot find any .exe or .zip files that might contain .exe files anywhere. I realize that there are instructions to compile binaries, but given the statement quoted above, I was hoping they'd been available - compiled, ready to go. Is that not the case?

**Comment by philipnbbc** [comment]
They are provided as part of the releases: https://github.com/bbc/bmx/releases. The Windows binary in the version 1.1 release is the [bmx-win64-binary-1.1.zip](https://github.com/bbc/bmx/releases/download/v1.1/bmx-win64-binary-1.1.zip) file. I'll fix the wording in the README as it's unclear and should simply reference the releases page. It so happens that I'll be making another release today if you want the latest.

**Comment by DJ8014** [comment]
Thank you! Clearly a lack of navigation knowledge on my part. Thanks again.
